site/content/docs/FAQs/_index.md

```diff
@@ -34,6 +46,43 @@ Files/directories are restored from the snapshots you create. To restore the dat
+
+Currently, encryption algorithms cannot be changed after a `repository` has been created.
+
+#### How Do I Enable Compression?
+
+Compression is controlled by [policies](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots) and is disabled by default. If compression is not working for a snapshot, it is likely that you have not enabled compression yet in your `policy`.
+
+Enabling compression when using `KopiaUI` is easy; edit the `policy` you want to add compression to and pick a `Compression Algorithm` in the `Compression` section. Kopia CLI users need to use the [`kopia policy set`](..reference/command-line/common/policy-set/) command as shown in the [Getting Started Guide](../getting-started/#policies). You can set compression on a per-source-directory basis...
+
+```shell
+kopia policy set </path/to/source/directory/> --compression=<deflate-best-compression|deflate-best-speed|deflate-default|gzip|gzip-best-compression|gzip-best-speed|pgzip|pgzip-best-compression|pgzip-best-speed|s2-better|s2-default|s2-parallel-4|s2-parallel-8|zstd|zstd-better-compression|zstd-fastest>
+```
+
+...or globally for all source directories:
+
+```shell
+kopia policy set --global --compression=<deflate-best-compression|deflate-best-speed|deflate-default|gzip|gzip-best-compression|gzip-best-speed|pgzip|pgzip-best-compression|pgzip-best-speed|s2-better|s2-default|s2-parallel-4|s2-parallel-8|zstd|zstd-better-compression|zstd-fastest>
+```
+If you enable or disable compression or change the compression algorithm, the new setting is applied going forward and not reteroactively. In other words, Kopia will not modify the compression for files/directories already uploaded to your repository.
+
+#### How Do I Enable Data Deduplication?
+
+[Data deduplication](../features/#backup-files-and-directories-using-snapshots) is enabled automatically by Kopia for all repositories, regardless of whether you use the GUI or CLI. You do not need to do anything.
+
+#### How Do I Change My Repository Password?
+
+You must use Kopia CLI if you want to change your `repository` password; changing password is not currently supported via Kopia GUI. The [`kopia repository change-password` command](../reference/command-line/common/repository-change-password/) is used to change your password. 
+
+Before changing your password, you must be [connected to your `repository`](../getting-started/#connecting-to-repository). This means that you **can** reset your password if you forget your password AND you are still connected to your `repository`. But this also means that you **cannot** reset your password if you forget your password and you are NOT still connected to your `repository`, because you will need your current password to connect to the `repository`.
+
+As a reminder, your `repository` password is used to [encrypt](../features/#end-to-end-zero-knowledge-encryption) your snapshots, so make sure to select a secure password!
```

Comment:
Technically, the password is used to derive a key to encrypt the repository primary encryption key (from which encryption keys are derived). However, that ends up being confusing.

What about:

```suggestion
Remember to select a secure _repository password_. The password is used to [decrypt](../features/#end-to-end-zero-knowledge-encryption) and access the data in your snapshots.
```